src/ci/docker/host-x86_64/mingw-check/Dockerfile

```
@@ -41,6 +41,10 @@ COPY host-x86_64/mingw-check/validate-error-codes.sh /scripts/
 ENV RUN_CHECK_WITH_PARALLEL_QUERIES 1
 
 ENV SCRIPT python3 ../x.py --stage 2 test src/tools/expand-yaml-anchors && \
+           # check library crates on all tier 1 targets
+           # We disable optimizing compiler built-ins because that requires building for the target
+           # We also skip the x86_64-unknown-linux-* targets as they're well tested by other jobs.
```

```suggestion
           # We also skip the x86_64-unknown-linux-gnu target as it is well tested by other jobs.
```
There's only one such target, is there?